src/legacy/ui/public/visualize/loader/pipeline_helpers/run_pipeline.ts

```diff
@@ -25,15 +25,15 @@ import { getInterpreter } from 'plugins/interpreter/interpreter';
 import { Adapters } from 'ui/inspector';
 import { Filters, Query, TimeRange } from 'ui/visualize';
 
-interface InitialContextObject {
+export interface InitialContextObject {
```

Comment:
They are used in the expressions service inside `data` to help with the local interim types there because atm there are not other types available. +1 for more complete expression types. As soon as your PR is merged I will fix the type references in the expressions service.